_posts/2024-07-22-bazel-q2-2024-community-update.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: posts
+title: "Bazel Q2 2024 Community Update"
+authors:
+ - karolinakalin
+ - keertk
+---
+
+## Announcements
+
+### BazelCon 2024
+
+The Bazel community, in partnership with The Linux Foundation, is excited to announce [BazelCon 2024](https://blog.bazel.build/2023/05/25/save-the-date-bazelcon2023.html), taking place on October 14-15 at the Computer History Museum in Mountain View, CA.
+
+Thanks to our sponsors, Google, BuildBuddy, EngFlow, AspectBuild, and Tweag, we invite you to join us for two days of insightful presentations, networking opportunities, and engaging discussions on all things Bazel. [Registration](https://events.linuxfoundation.org/bazelcon/) is now open.
```

Comment:
The list of sponsors on the official page is different (VirtusLab instead of Tweag), and IIRC there will be a seventh (Gold) sponsor.